src/Components/TextSquareBadge/TextSquareBadge.jsx

```diff
@@ -0,0 +1,5 @@
+export const TextBadgeSquare = (props) => {
+    return (
+        <span className={`text-badge ${props.className}`}>{props.text}</span>
```

Comment:
nice use of classname, didnt think one could access it like that

Reply:
Thanks